shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1228,6 +1232,11 @@ void updateSemantics(@NonNull ByteBuffer buffer, @NonNull String[] strings) {
             }
             if (semanticsNode.hasFlag(Flag.IS_FOCUSED)) {
                 inputFocusedSemanticsNode = semanticsNode;
+                AccessibilityEvent event = obtainAccessibilityEvent(
+                    inputFocusedSemanticsNode.id,
+                    AccessibilityEvent.TYPE_VIEW_FOCUSED
```

Comment:
There's no corresponding unfocused event, right?

Reply:
No, not that I know of.  I assume that it just assumes a single item can be focused.  There is one for clearing the accessibility focus, but I don't think we want to mess with that one.